### PR TITLE
move strings into constants

### PR DESCRIPTION
**Pull request by BertKleewein**

### CODE REVIEW

File: python/common/thief_constants.py [hunk 1]
```diff
@@ -3,7 +3,91 @@
...
+    JSON_TYPE_AND_ENCODING = {
+        "contentType": "application/json",
+        "contentEncoding": "utf-8",
+    }
+    PROPERTIES = "properties"
+    REPORTED = "reported"
+
+
+class Fields(object):
+    class Deprecated(object):
```
Comment (BertKleewein) [inline python/common/thief_constants.py:18]: >Deprecated [](start = 10, length = 10)

I'm going to remove the functionality around these strings, but I wanted to do that as a separate PR.

File: python/device/device.py [hunk 2]
```diff
@@ -310,7 +314,7 @@ def get_session_metrics(self):
             "runTime": str(self.metrics.run_time),
             "runState": str(self.metrics.run_state),
             "exitReason": self.metrics.exit_reason,
-            "pairingId": self.pairing_id,
+            Fields.Deprecated.PAIRING_ID: self.pairing_id,
```
Comment (BertKleewein) [inline python/device/device.py:317]: I'm going to move other strings into thief_constants.py, but not in this change.

File: python/common/thief_constants.py [hunk 3]
```diff
@@ -3,7 +3,91 @@
...
 
-class ServiceAckType(object):
-    TELEMETRY_SERVICE_ACK = "telemetry"
-    ADD_REPORTED_PROPERTY_SERVICE_ACK = "add_reported"
-    REMOVE_REPORTED_PROPERTY_SERVICE_ACK = "remove_reported"
+class Const(object):
+    JSON_CONTENT_TYPE = "application/json"
+    JSON_CONTENT_ENCODING = "utf-8"
+    JSON_TYPE_AND_ENCODING = {
+        "contentType": "application/json",
```
Comment (drwill-ms) [inline python/common/thief_constants.py:10]: Can this value reference JSON_CONTENT_TYPE?

File: python/common/thief_constants.py [hunk 4]
```diff
@@ -3,7 +3,91 @@
...
+        CONFIG = "config"
+
+        PAIRING = "pairing"
+
+        class Pairing(object):
+            REQUESTED_SERVICE_POOL = "requestedServicePool"
+            SERVICE_RUN_ID = "serviceRunId"
+            DEVICE_RUN_ID = "deviceRunId"
+
+        TEST_CONTENT = "testContent"
```
Comment (drwill-ms) [inline python/common/thief_constants.py:46]: Is this a stand-alone constant? I.E. not in a class?

File: python/device/device.py [hunk 5]
```diff
@@ -635,7 +641,7 @@ def send_metrics_to_azure_monitor(self, props):
         """
         # we don't record session_metrics to azure monitor
 
-        system_health_metrics = props["systemHealthMetrics"]
+        system_health_metrics = props[Fields.Reported.SYSTEM_HEALTH_METRICS]
         self.reporter.set_process_cpu_percent(system_health_metrics["processCpuPercent"])
```
Comment (drwill-ms) [inline python/device/device.py:645]: TODO: "processCpuPercent" and the others below?

**Review by drwill-ms — APPROVED** [review]

File: python/common/thief_constants.py (hunk 3 above)
Comment (BertKleewein) [reply python/common/thief_constants.py:10]: not cleanly.  There's a few different ways to hack it, but they would only add confusion.
